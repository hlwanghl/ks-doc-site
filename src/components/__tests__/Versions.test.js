import React from 'react'
import renderer from 'react-test-renderer'
import Versions from '../Versions'

describe('Versions', () => {
  it('renders correctly', () => {
    const props = {
      versions: [
        {
          label: 'Express v1.0.0-alpha',
          value: 'express',
        },
        {
          label: 'Advance v1.0.0',
          value: 'advance-v1.0.0',
        },
      ]
    }

    const tree = renderer.create(<Versions {...props} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
