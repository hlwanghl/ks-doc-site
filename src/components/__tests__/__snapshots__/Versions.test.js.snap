// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Versions renders correctly 1`] = `
<div
  className="sc-bdVaJa fLDVnG"
>
  <div
    className="sc-bwzfXH fstIjj"
    onClick={[Function]}
  >
    <SvgMock
      className="version-logo"
    />
    <div
      className="version-text"
    >
      <SvgMock
        className="kubesphere-icon"
      />
      <p />
    </div>
    <SvgMock
      className="version-arrow"
    />
  </div>
  <div>
    <div
      className="sc-htpNat bOULPv"
    >
      <div
        className="sc-bxivhb gqabTH"
        onClick={[Function]}
        selected={false}
      >
        Express v1.0.0-alpha
      </div>
      <div
        className="sc-bxivhb gqabTH"
        onClick={[Function]}
        selected={false}
      >
        Advance v1.0.0
      </div>
    </div>
  </div>
</div>
`;
